src/subcommand.rs

```diff
@@ -485,12 +487,37 @@ impl Subcommand {
         let mut line_width = UnicodeWidthStr::width(*name);
 
         for parameter in &recipe.parameters {
-          if let Some(evaluator) = &mut evaluator {
-            let parameter = parameter.evaluate_default(evaluator)?;
-          };
-          line_width += UnicodeWidthStr::width(
-            format!(" {}", parameter.color_display(Color::never())).as_str(),
-          );
+          // FIXME: unwrap -> ?
+          if let Some(evaluated) = evaluator
+            .as_mut()
+            .map(|evaluator| parameter.evaluate_default(evaluator))
+            .transpose()
+            .unwrap()
+            .flatten()
```

Comment:
The `unwrap` on L495 is a result of what I previously wrote, I have no idea why it thinks that we're borrowing from the locally owned `justfile` when returning an eval error.
So I implemented the functionality anyway by unwrapping and left the `FIXME` in there.